generic_permissions/visibilities.py

```diff
@@ -88,6 +128,17 @@ def get_attribute(self, instance):
 
         return model_instance
 
+    def bind(self, field_name, parent):
+        parent_cls = type(parent)
+        if parent_cls is VisibilityManyRelatedField:
+            return super().bind(field_name, parent)
+
+        if VisibilitySerializerMixin not in parent_cls.mro():
```

Comment:
Just thinking about it a bit more, you could use `isinstance()` here as well:

```suggestion
        if not isinstance(parent, VisibilitySerializerMixin):
```

It's basically the same: `isinstance()` just checks the method resolution order (MRO) of the associated class

Reply:
Done